CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Unreleased
 
+## JavaScript
+- Fixed a bug where if any injected dependencies to a JavaScript (e.g. `ux:Name`, `ux:Dependency`, `dep:`) have names that correspond to EcmaScript 5.1 reserved words, these dependencies will now be available with a dollar sign `$` prefix (e.g. `$undefined`) instead of causing a script compile error.
```

Comment:
Would it be possible to *always* make a dependency named `foo` available as `$foo` as well, so people don't need to manually keep track of the current list of reserved keywords?